### ISSUE TITLE
Output formatting attempts to apply to header row

### ISSUE THREAD

**Issue by kmehkeri** [harelba/q#89]
When I use '-O' option to output header it seems q attempts to apply formatting to it, causing exception.

```
$ q -H -O -d',' -f'3=%011d' "SELECT * FROM SELLOUT_SO_BG"
Traceback (most recent call last):
  File "./q", line 1639, in <module>
    run_standalone()
  File "./q", line 1628, in run_standalone
    q_output_printer.print_output(STDOUT,sys.stderr,q_output)
  File "./q", line 1319, in print_output
    self._print_output(f_out,f_err,results)
  File "./q", line 1366, in _print_output
    row_str.append(fmt_str % self.output_field_quoting_func(self.output_params.delimiter,col))
TypeError: %d format: a number is required, not unicode
```

Removing either '-O' or '-f' makes it work.

**Comment by harelba** [comment]
Hi @kmehkeri,

Thanks for opening the bug. It is a bug. I've fixed it and pushed the fix into the master branch. It will be released as part of the next version.

In the mean time you can take the q executable from [here](https://cdn.rawgit.com/harelba/q/master/bin/q), or send me your email address and I'll send you the executable.

Please tell me if there are still any issues with the newer executable, of if you encounter any others.
Harel

I'll close the issue after the next version is released.

**Comment by kmehkeri** [comment]
It works fine now. Thanks for the quick fix and a great tool!

**Comment by harelba** [comment]
Fixed in v1.6.3
